aion/web/elm/src/General/Constants.elm

```diff
@@ -19,18 +19,18 @@ gameCardConfig location =
 panelCardConfig : Location -> SimpleCardConfig
 panelCardConfig location =
     { svgImage = (host location) ++ "svg/tasks.svg"
-    , title = "Panel"
-    , description = "Create new rooms, new categories and new questions."
+    , title = "Workspace"
+    , description = "Create new rooms, add new categories and new questions."
     , url = panelPath
-    , buttonText = "Visit panel"
+    , buttonText = "Visit workspace"
```

Comment:
All our resources in Elm code are put in a `/panel` directory and now we are changing this nomenclature in UX but not in the code.

Reply:
Yes, I know. This is also why I'm not for introducing naming 'just for the dev process' because later it requires not only changes in UI, but also in the whole code. 

Anyway since you've given this feature a name 'panel', I think we could stick to using that in code and only switch the UI naming. We could also think about a terminal name for this feature and introduce it now everywhere - both in code and UI (but we may also do that after releasing v1, as we are mostly caring about releasing it asap now and will do the elm refactor later anyway).

Reply:
Let's merge this PR and we can address this in Elm refactor.

Reply:
Ack